data/categories.txt

```diff
@@ -3796,7 +3817,7 @@ zh-Hant:有毒垃圾|廢棄電池處理|有害垃圾
 fa:پسماند خطرناک|بازیافت باتری|پسماند سمی
 
 recycling-clothes|@recycling
-en:Old clothes|Recycle old clothes
+en:Clothes
```

Comment:
It's a recycling center which accepts clothes, and probably may be confused with a Clothes shop. See https://wiki.openstreetmap.org/wiki/Tag:amenity%3Drecycling

Reply:
Agree. The same goes for all these recycling entries. I've changed this to "Clothes Recycling", and changed "Glass" below to "Glass Waste". I've also "ucwords" capitalised them all, as is the requested style.